hbw/app/javascript/packs/hbw/components/form/group.js.jsx

```diff
@@ -123,6 +124,10 @@ modulejs.define('HBWFormGroup', ['React', 'HBWFormDatetime',
     notSerializableFields = () => ['static'];
 
     serialize = () => {
+      if (this.props.hidden || this.props.disabled) {
```

Comment:
`hidden` prop only.
`disabled` prop is delegated to nested components.

Reply:
Done